### DISCUSSION TITLE
Temporal upscaler not working in RE2 with latest DLSS files, PD-Upsacler and UpsaclerBasePlugin

### DISCUSSION THREAD

**Discussion by Header2000** [praydog/REFramework#1170] [General Questions]
Hi Guys, 

Just got my Meta Quest 3 recently and wanted to check out the RE2 VR mod, following the instructions on youtube and on this site, I was able to install the new pd upsacler mod. I also downloaded the latest DLSS and UpsaclerBasePlugin, put the 
- dinput8.dlldinput8.dll (Re2 File for DX12 Version)
- nvngx_dlss.dll
- PDPerfPlugin.dll
all in the game base directory were the RE2.exe file is. 

When loading the game, I also have the TemporalUpscaler menu, but with an error message: "Backend is not loaded, temporal upscaler will not work. Make sure you have downloaded the upsaclerBasePlugin and the corresponding DLLs for your preferred upsacler. 
I Have tried all the soulutions here on the github mentioned, is it possible that the latest build is broken? Does anyone else having this problem as well.
Any help in solving this problem is appreciated.

Sources for the files I used
UpscalerBasePlugin: https://www.nexusmods.com/site/mods/502?tab=files&file_id=4445
DLSS files: https://www.techpowerup.com/download/nvidia-dlss-dll/
PD-Upscaler: https://github.com/praydog/REFramework/actions/runs/11770014195

PC: Win 11
RE2 latest  uptodate Version 22.12.24
GFX 2070 Super 
AM5 with AMD Ryzen 7 7800X3D 
32 gigs of RAM

**Comment by felixreed** [comment]
You need version 1.1.2 of UpscalerBasePlugin. 
UpscalerBasePlugin was recently updated and PD-Upscaler may not be adapted yet.

**Reply by Header2000** [reply]
Thanks for the answer, I found version 1.1.2 of the plugin on this website: https://allmods.net/starfield-mods/starfield-miscellaneous/upscaler-base-plugin-v1-1-2/ but even with this version the problem still exists, so it still tells me the backend is not loaded, temporal upscaler will not work! Do you have a version that works with the released PD-Upscaler version? Or do you know which version of the Upscale base plugin and PD-Upscaler work together without problems?

Maybe you have a link to a version that works?

**Reply by felixreed** [reply]
https://www.nexusmods.com/site/mods/502?tab=files&file_id=2293f
![image](https://github.com/user-attachments/assets/b614e21d-3262-4a1a-b3bf-f0f19239140c)
![VirtualDesktop Android-20241228-160518](https://github.com/user-attachments/assets/92373216-1696-4d2a-a27d-429b314efa0a)

**Reply by Header2000** [reply]
Thank you Hinlavic , that did the trick. DLSS now works for me. Hope that others that have similar problems this thread is a help. :) 

Best Regards

**Reply by bigloop123** [reply]
I still cant see the upscaler in the menu...:(

**Reply by Shijiale111** [reply]
> Thank you Hinlavic , that did the trick. DLSS now works for me. Hope that others that have similar problems this thread is a help. :)
> 
> Best Regards

Hey mate,

May I know how you solved the issue? I've been trying to run dlss on re4r and village, but it just doesn't want to work. Got your same problem.
I've tried combining 10 different versions of reframework pd upscaler, including nightly builds, with upscaler base plugin 1.1.2, but it just doesn't work. 
Wasted hours on this and gave up.

Is it possible that me running the latest Proton-GE 10 runner might cause compatibility problems?
Or could you tell me which versions work together?

Thanks a lot for your reply in advance,

Gabriel

**Reply by Plejer01** [reply]
Me too did someone know how to fix it ?

**Reply by TiSimpson1969** [reply]
I gave up. Got RE4 working great with DLSS though. Same Error Temporal Upscaling shows it didn't load backend. Tried all versions of PDupscaler including 1.1.2 and all versions of nvngx DLSS dll as well

**Reply by Andu21x** [reply]
how did you get RE4 (remake) with DLSS?

**Comment by YuryNiakhai** [comment]
still doesn't work
